packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -548,16 +548,12 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   /// Used by [EdgeDraggingAutoScroller] to progress the position forward when a
   /// drag gesture reaches the edge of the [Viewport].
   Offset get deltaToScrollOrigin {
-    switch (axisDirection) {
-      case AxisDirection.down:
-        return Offset(0, position.pixels);
-      case AxisDirection.up:
-        return Offset(0, -position.pixels);
-      case AxisDirection.left:
-        return Offset(-position.pixels, 0);
-      case AxisDirection.right:
-        return Offset(position.pixels, 0);
-    }
+    return switch (axisDirection) {
+      AxisDirection.down  => Offset(0,  position.pixels),
+      AxisDirection.up    => Offset(0, -position.pixels),
+      AxisDirection.left  => Offset(-position.pixels, 0),
+      AxisDirection.right => Offset( position.pixels, 0),
```

Comment:
This looks a bit funky to me, consider instead:

```suggestion
      AxisDirection.right => Offset(position.pixels,  0),
```

Or:

```suggestion
      AxisDirection.right => Offset(position.pixels, 0),
```